airflow/api_connexion/endpoints/connection_endpoint.py

```diff
@@ -63,11 +63,15 @@ def get_connection(connection_id, session):
 @security.requires_access([(permissions.ACTION_CAN_READ, permissions.RESOURCE_CONNECTION)])
 @format_parameters({'limit': check_limit})
 @provide_session
-def get_connections(session, limit, offset=0):
+def get_connections(session, limit, offset=0, order_by="id"):
```

Comment:
Should we sort by `id` or `conn_id`  -- no storng preference

Reply:
It was originally ordered by `id`, so I made that the default else it won't be backward compatible

Reply:
sounds good.